components/server/src/workspace/workspace-service.ts

```diff
@@ -613,7 +617,24 @@ export class WorkspaceService {
         });
     }
 
-    public async getSupportedWorkspaceClasses(userId: string): Promise<SupportedWorkspaceClass[]> {
+    public async getSupportedWorkspaceClasses(user: User): Promise<SupportedWorkspaceClass[]> {
```

Comment:
we don't need to change the argument to pass the whole user, if all we want is checking the feature flag. you can pass in a `{ id: string }` as a user